src/domUpdates.js

```diff
@@ -1,29 +1,292 @@
+    </div>
+  `;
+};
+
+// event handlers and functions dependent user interation👇
+export const renderTripsGrid = (e) => {
+  cardsGrid.classList.add('cards-grid');
+  console.log(currentTraveler);
+  mainDisplay.removeEventListener('click', displayBookTripPage);
+  let tripType = e.target.id;
+  if (
+    e.target.id === 'upcomingTrips' ||
+    e.target.id === 'pastTrips' ||
+    e.target.id === 'pendingTrips'
+  ) {
+    let headerDisplay = e.target.innerText;
+    mainHeader.innerText = `${headerDisplay}`;
+    cardsGrid.innerHTML = '';
+    currentTraveler[tripType].forEach((trip) => {
+      let startDate = dayjs(trip.date).format('MM/DD/YYYY');
+      let endDate = dayjs(startDate)
+        .add(trip.duration, 'day')
+        .format('MM/DD/YYYY');
+      cardsGrid.innerHTML += `
+      <section class="card">
+      <div class="destination-card">
+        <img class="destination-image" src="${trip.tripsDestination.image}" alt="${trip.tripsDestination.alt}">
+        <div class="destination-details">
+          <h4 class="city">${trip.tripsDestination.destination}</h4>
+          <p class="lodging-cost">Number of travelers: ${trip.travelers}</p>
+          <p class="lodging-cost">Travel dates: ${startDate} - ${endDate}</p>
+          <p class="flight-cost">Trip cost: ${trip.cost}</p>
+        </div>
+      </div>
+      `;
+    });
+  }
+};
+
+export const displayBookTripPage = (e) => {
+  if (typeof eval(e.target.id) === 'number') {
+    const destinationID = parseInt(e.target.id);
+    destinationSelected = destinationsData.find(
+      (destinationData) => destinationData.id === destinationID
+    );
+
+    mainDisplay.removeEventListener('click', displayBookTripPage);
+
+    mainHeader.innerText = 'Plan Your Trip';
+    cardsGrid.innerHTML = '';
+    cardsGrid.classList.remove('cards-grid');
+    cardsGrid.innerHTML += `
+    <div class="form-and-destination"
+      <section class="selected-destination-card" id=${destinationSelected.id}>
+        <div class="destination-card">
+          <img class="destination-image" id=${destinationSelected.id} name="${
+      destinationSelected.destination
+    }" src="${destinationSelected.image}" alt="${destinationSelected.alt}">
+          <div class="destination-details">
+            <h4 class="city">${destinationSelected.destination}</h4>
+            <p class="lodging-cost">Estimated lodging: $${
+              destinationSelected.estimatedLodgingCostPerDay
+            } / night</p>
+            <p class="flight-cost">Estimated roundtrip flight: $${
+              destinationSelected.estimatedFlightCostPerPerson
+            }</p>
+          </div>
+        </div>
+      </section>
+      <section class="trip-form" id="tripForm">
+        <label>Departure:
+          <input type="date" id="start" name="trip"
+          placeholder="${dayjs().format('YYYY-MM-DD')}"
+          min="${dayjs().format('YYYY-MM-DD')}" max="${dayjs()
+      .add(1, 'year')
+      .format('YYYY-MM-DD')}" required>
```

Comment:
I think setting type="date" for <input> sets the placeholder for you, but I'll have to double check to be sure.